easybuild/easyblocks/w/wrf.py

```diff
@@ -362,8 +369,10 @@ def install_step(self):
     def sanity_check_step(self):
         """Custom sanity check for WRF."""
 
-        mainver = self.version.split('.')[0]
-        self.wrfsubdir = "WRFV%s" % mainver
+        if LooseVersion(self.version) < LooseVersion('4.0'):
+            self.wrfsubdir = "WRFV3"
+        else:
+            self.wrfsubdir = "WRF-%s" % self.version
```

Comment:
Move the self.wrfsubdir setting to "init" instead of repeating the code. And use it in the setting of wrfdir above. Cleaner and less code.

Reply:
@andreas-h I took a look at this myself + some more cleanup for these easyblocks, please take a look at https://github.com/andreas-h/easybuild-easyblocks/pull/4

Reply:
Thanks a lot :)